src/create-card.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const styles = {
+  background: {
+    backgroundColor: '#fffbe0',
```

Comment:
Stick with the default colors provided by Bootstrap. White is fine. Try to make your styles look like the Bootstrap docs. They already put a lot of thought into the colors and whitespace.